### PR TITLE
🏀 Buzzer beaters

### LINKED ISSUE #708: 'Copied' button lingers a bit to long
Should either disappear when you close pop-over or clicking 'copy' should close pop-over

https://user-images.githubusercontent.com/7005061/146468985-146e7f8e-fd11-4dee-bda0-a24adc7ffe66.mov

### LINKED ISSUE #709: 'Import account' page typo
### Discord Discussion Link

_No response_

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

-

### What did not work?

"copy & paste" - & is missing

Screen Shot 2021-12-16 at 6 17 49 PM

### Version

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by henryboldi**
- [x] Tiny tweak to show 0 balance if none exist
- [x] Trim recovery phrase before testing if valid
- [x] Fixes #709
- [x] Fixes #708
- [x] Adds a background color fallback for if an ENS profile avatar URL doesn't load
- [x] Add explanation on activity loading, remove spinner

activity loading 1

activity loading 2

### CODE REVIEW

File: ui/components/Wallet/WalletAccountBalanceControl.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default function WalletAccountBalanceControl(
             })}
           >
             <span className="dollar_sign">$</span>
-            {balance}
+            {balance || 0}
```
Comment (Shadowfiend) [inline ui/components/Wallet/WalletAccountBalanceControl.tsx:85]: ?? plz

File: ui/pages/Onboarding/OnboardingImportMetamask.tsx [hunk 2]
```diff
@@ -84,9 +84,10 @@ export default function OnboardingImportMetamask(props: Props): ReactElement {
   }, [history, areKeyringsUnlocked, keyringImport, nextPage, isImporting])
 
   const importWallet = useCallback(async () => {
-    if (isValidMnemonic(recoveryPhrase)) {
+    const trimmedRecoveryPhrase = recoveryPhrase.trim()
```
Comment (Shadowfiend) [inline ui/pages/Onboarding/OnboardingImportMetamask.tsx:87]: 🎉

File: ui/components/Wallet/WalletAccountBalanceControl.tsx (hunk 1 above)
Comment (henryboldi) [reply ui/components/Wallet/WalletAccountBalanceControl.tsx]: ✅ :)

File: ui/pages/SignTransaction.tsx [hunk 3]
```diff
@@ -219,7 +219,7 @@ export default function SignTransaction(): ReactElement {
           iconSize="large"
           size="large"
           type="secondary"
-          onClick={() => history.goBack()}
+          onClick={() => window.close()}
```
Comment (Shadowfiend) [inline ui/pages/SignTransaction.tsx:222]: Ohp, no. This component is used for regular signing in the send flow as well 😬 

I think reject will need to call an async thunk in redux and emit an event, we'll need to hook it up to a rejection pipeline in the internal Ethereum provider (which will eventually cause the popover close in the bridge service), and then we can leave the history.goBack bit for non-dapp signing and have that happen after the thunk dispatch.

**Review by Shadowfiend — DISMISSED** [review]
No go on the reject fix I think.

File: ui/pages/SignTransaction.tsx (hunk 3 above)
Comment (Shadowfiend) [reply ui/pages/SignTransaction.tsx:222]: I should be able to snipe this tomorrow; it sounds like a lot but it's actually just a few loc in several places.

File: ui/components/Shared/SharedCurrentAccountInformation.tsx [hunk 4]
```diff
@@ -22,9 +22,9 @@ export default function SharedCurrentAccountInformation(): ReactElement {
             border-radius: 12px;
             width: 32px;
             height: 32px;
-            background-color: white;
             margin-left: 8px;
             background: url("${avatarURL ?? "./images/portrait.png"}");
+            background-color: var(--green-40);
```
Comment (greg-nagy) [inline ui/components/Shared/SharedCurrentAccountInformation.tsx:27]: 🧡

File: ui/components/Snackbar/Snackbar.tsx [hunk 5]
```diff
@@ -10,7 +10,7 @@ import { useBackgroundSelector } from "../../hooks"
 export default function Snackbar(): ReactElement {
   const dispatch = useDispatch()
   const snackbarMessage = useBackgroundSelector(selectSnackbarMessage)
-  const msTillDismiss = 5000
+  const msTillDismiss = 2500
```
Comment (greg-nagy) [inline ui/components/Snackbar/Snackbar.tsx:13]: The length of the snackbar being shown does not seem to behave according to this value.

CleanShot 2021-12-17 at 18 21 15

Comment (greg-nagy) [reply ui/components/Snackbar/Snackbar.tsx:13]: A possible workaround is to handle the timeout locally and change the `open` class based on the local value.

Comment (henryboldi) [reply ui/components/Snackbar/Snackbar.tsx:13]: Perhaps it's using the old value somehow? It appears to be working on my end

**Review by greg-nagy — APPROVED** [review]
This is good to go. 

The timeout timing seems to be strange, but I don't think it's a blocker.

File: ui/components/Snackbar/Snackbar.tsx (hunk 5 above)
Comment (greg-nagy) [reply ui/components/Snackbar/Snackbar.tsx:13]: It was a glitch on my machine :) It's working now!

**Pull request merged**